src/OrleansRuntime/Streams/PubSub/PubSubRendezvousGrain.cs

```diff
@@ -56,8 +55,7 @@ static PubSubRendezvousGrain()
                     await ClearStateAsync(); //State contains no producers or consumers, remove it from storage
             }
 
-            if (logger.IsVerbose)
-                logger.Info("OnActivateAsync-Done");
+            logger.Info("OnActivateAsync-Done");
```

Comment:
I don't think we want this to be Info. It used to log only under Verbose. the usual rule of thumb is to log at Info only silo level global events, and not individual grain events, which are logged at Verbose. Individual application msgs are at Verbose 2. We don't always strictly follow this rule, but we try to.